translations.json

```diff
@@ -392,7 +392,7 @@
             "en": "WARNING: Experimental options below - If you touch these, you confirm that you can troubleshoot any issues you encounter yourself and won't ask for help.",
             "fr-FR": "AVERTISSEMENT : Options expérimentales ci-dessous - Si vous y touchez, vous confirmez que vous pouvez résoudre vous-même les problèmes que vous rencontrez et que vous ne demanderez pas d'aide.",
             "ko-KR": "주의: 아래 옵션들은 실험적인 기능입니다. - 아래 옵션을 사용할 때, 마주치는 모든 문제를 직접 해결할 수 있고 도움을 요청하지 않을 것을 확인하고 사용하세요.",
-            "zh-CN": "警告：以下选项是试验性功能———一旦启用，视为您确认可以自己解决遇到的任何问题，而不会寻求帮助。"
+            "zh-CN": "警告: 以下选项是试验性功能————一旦启用, 视为你确认可以自己解决遇到的任何问题, 且不会寻求帮助。"
```

Comment:
It is barely to use dash in Chinese, you can re-word this or just remove it. In most case that don't infect the meaning of the sentences.